_posts/2020-02-17-Combining-MongoDB-and-AWS-Lambda.md

```diff
@@ -0,0 +1,310 @@
+---
+layout: post
+authors: [nick_van_hoof]
+title: 'Marrying MongoDB Atlas and AWS Lambda'
+image: /img/2020-02-17-Combining-MongoDB-Atlas-and-AWS-Lambda/featured-image.png
+tags: [MongoDB, Cloud, AWS, AWS Lambda, Serverless]
+category: Cloud
+comments: false
+---
+
+### Reading time: 10 min 30 sec
+
+# Table of contents
+* [MongoDB and AWS Lambda: a successful marriage?](#mongodb-and-aws-lambda-a-successful-marriage)
+* [MongoDB and AWS Lambda: Why?](#mongodb-and-aws-lambda-why)
+* [MongoDB and AWS Lambda: Performance](#mongodb-and-aws-lambda-performance)
```

Comment:
A bit too much repetition for my taste, can you drop the "MongoDB and AWS Lambda:" for the last 2 titles?

Reply:
Yes, will do.